webapp/resources/templates/casU2fLoginView.html

```diff
@@ -42,8 +42,8 @@
 <main role="main" class="container mt-3 mb-3">
     <div layout:fragment="content" id="login">
         <div class="alert alert-info">
-            <h3 th:text="cas.mfa.u2f.authentication.device">Authenticate Device</h3>
-            <div th:utext="cas.mfa.u2f.authentication.device.message" th:replace="tag">
+            <h3 th:text="#{cas.mfa.u2f.authentication.device}">Authenticate Device</h3>
+            <div th:utext="#{cas.mfa.u2f.authentication.message}">
```

Comment:
@daramos Any reason for removing the th:replace="tag" attribute?  Same for the one below.

Reply:
Maybe because there is this thymeleaf exception : 
ERROR [org.thymeleaf.TemplateEngine] - <[THYMELEAF][ajp-nio-8109-exec-4] Exception processing template "error": Error resolving template "tag", template might not exist or might not be accessible by any of the configured Template Resolvers (template: "casU2fLoginView" - line 46, col 71)>
org.thymeleaf.exceptions.TemplateInputException: Error resolving template "tag", template might not exist or might not be accessible by any of the configured Template Resolvers (template: "casU2fLoginView" - line 46, col 71)
	at org.thymeleaf.engine.TemplateManager.resolveTemplate(TemplateManager.java:870) ~[thymeleaf-3.0.9.RELEASE.jar:3.0.9.RELEASE]
	at org.thymeleaf.engine.TemplateManager.parseStandalone(TemplateManager.java:250) ~[thymeleaf-3.0.9.RELEASE.jar:3.0.9.RELEASE]